libs/auth/src/lib/auth.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core'
+import { MeApiService } from '@lib/gn-api'
+import { map, shareReplay } from 'rxjs/operators'
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  constructor(private meApi: MeApiService) {}
+
+  authReady() {
+    return this.meApi.getMe().pipe(
```

Comment:
Don't we want to store user info return by the `/me` service ?
Maybe it's out of scope now.

Reply:
exactly, for now we just want to be sure the browser received the correct cookies